packages/firebase_ml/README.md

```diff
@@ -1 +1,14 @@
-# Firebase Machine Learning
+# firebaseml
+
+A new Flutter plugin.
```

Comment:
let's document the difference between firebase-ml and firebase-ml-vision too

Reply:
Done